setup.py

```diff
@@ -2,11 +2,9 @@
 
 from setuptools import setup
 
-import i18n
-
 setup(
     name='edx-i18n-tools',
-    version=i18n.__version__,
+    version='0.3.7',
```

Comment:
It's that way in the cookiecutter, but they don't import the product code into setup.py.  When I moved the import of config, it meant that config was being imported when setup.py ran.  Which meant the package couldn't be installed because setup.py imported __init__.py, which imported config.py which tried to import yaml, but that wasn't installed yet because setup.py hadn't run.